qemu/tests/cfg/netkvm_buffer_shortage.cfg

```diff
@@ -0,0 +1,31 @@
+- netkvm_buffer_shortage:
+    virt_test_type = qemu
+    type = netkvm_buffer_shortage
+    only Windows
+    only virtio_net
+    vhost = on
+    timeout = 360
+    cdroms += " virtio"
+    vms += " vm2"
+    image_snapshot = yes
+    start_vm = yes
+    start_vm_vm2 = no
+    smp = 2
+    queues = ${smp}
+    vectors = 1024
+    port_num = 12345
+    copy_dest = "C:\"
+    server_script = "server.py"
+    client_script = "client.py"
+    check_live_python = "tasklist | findstr /i python"
+    copy_all_cmd = 'xcopy "WIN_UTILS:\packet_loss_scripts\*" ${copy_dest}'
+    i386:
+        psutil_whl = "psutil-6.1.1-cp37-abi3-win32.whl"
+    x86_64:
+        psutil_whl = "psutil-6.1.1-cp37-abi3-win_amd64.whl"
+    pip_cmd = "py -m pip install ${psutil_whl}"
+    dest_location = "pushd ${copy_dest}"
```

Comment:
Can we use the absolute path directly so that we don't have to switch directories again and again?

Reply:
Okay, if so, that's fine with me.